src/cartridges/int_adyen_SFRA/cartridge/templates/default/checkout/billing/adyenComponentForm.isml

```diff
@@ -2,65 +2,108 @@
 <link rel="stylesheet" type="text/css" href="${pdict.AdyenHelper.getCheckoutCSS()}"/>
 <isset name="adyenRecurringPaymentsEnabled" value="${pdict.AdyenHelper.getAdyenRecurringPaymentsEnabled()}" scope="page"/>
 <script src="https://pay.google.com/gp/p/js/pay.js"></script>
+<isset name="ShowConfirmationPaymentFromComponent" value="${URLUtils.https('Adyen-ShowConfirmationPaymentFromComponent')}" scope="page"/>
+
 <isscript>
     var assets = require('*/cartridge/scripts/assets.js');
     assets.addJs('/js/bundle.js');
 </isscript>
-<script>
-    window.Configuration = {
-        locale: '${request.locale}',
-        originKey: '${pdict.adyen.originKey}',
-        environment: '${pdict.adyen.environment}'
-    };
-    window.installments = '${pdict.adyen.installments}';
-    window.paypalMerchantID = '${pdict.adyen.paypalMerchantID}';
-    window.googleMerchantID = '${pdict.adyen.googleMerchantID}';
-    window.cardholderNameBool = '${pdict.adyen.cardholderNameBool}';
-    window.paypalIntent = '${pdict.adyen.paypalIntent}';
-    window.merchantAccount = '${pdict.adyen.merchantAccount}';
-    var showStoreDetails = ${customer.authenticated && adyenRecurringPaymentsEnabled};
 
-    window.getPaymentMethodsURL = "${URLUtils.https('Adyen-GetPaymentMethods')}";
-    window.paymentFromComponentURL = "${URLUtils.https('Adyen-PaymentFromComponent')}";
+<script>
+      window.Configuration = {
+          locale: '${request.locale}',
+          originKey: '${pdict.adyen.originKey}',
+          environment: '${pdict.adyen.environment}',
+      };
+      window.adyenClientKey = '${pdict.adyen.adyenClientKey}';
+      window.amazonMerchantID = '${pdict.adyen.amazonMerchantID}';
+      window.amazonStoreID = '${pdict.adyen.amazonStoreID}';
+      window.amazonPublicKeyID = '${pdict.adyen.amazonPublicKeyID}';
 </script>
 
-<isset name="AdyenHelper" value="${require('*/cartridge/scripts/util/adyenHelper')}" scope="pdict"/>
-<isset name="ratePayValues" value="${pdict.AdyenHelper.getRatePayID()}" scope="page"/>
+<isif condition="${request.getHttpQueryString().indexOf('amazonCheckoutSessionId') > -1} ">
+    <link rel="stylesheet" type="text/css" href="${pdict.AdyenHelper.getCheckoutCSS()}"/>
+    <isscript>
+        var assets = require('*/cartridge/scripts/assets.js');
+        assets.addJs('/js/amazon.js');
+    </isscript>
```

Comment:
Just make this one script. line 31 opens another script tag

Reply:
There is a difference between <isscript> and <script>